packages/react/src/components/SheetOverlay/ContentEditable.tsx

```diff
@@ -13,70 +13,78 @@ type ContentEditableProps = Omit<
   allowEdit?: boolean;
 };
 
-class ContentEditable extends React.Component<ContentEditableProps> {
-  lastHtml?: string = undefined;
+const ContentEditable = ({ ...props }: ContentEditableProps) => {
```

Comment:
ContentEditable:React.FC<ContentEditableProps>

Reply:
thank you.
fixed

Reply:
In additionl, using React.FC is for unified code style, and it seems that there is no need to use it, but from experience, you may not have any huge performance implications of not using it.

Reply:
yeah. I see that. That's correct. thank you.